app/config/FrontendAppConfig.scala

```diff
@@ -39,6 +39,8 @@ class FrontendAppConfig @Inject() (configuration: Configuration, servicesConfig:
   lazy val betaFeedbackUrl = s"$contactHost/contact/beta-feedback"
   lazy val betaFeedbackUnauthenticatedUrl = s"$contactHost/contact/beta-feedback-unauthenticated"
 
+  lazy val signOutUrl: String = "/sign-out"
```

Comment:
Why we need this ?

Reply:
We don't! Good catch - Removed in https://github.com/hmrc/voa-bar-xml-frontend/pull/136/commits/a17815b8dfb94b945a77d6663fd593fb147505da